index.html

```diff
@@ -142,97 +157,94 @@ <h2 class="datatext">Profesional Art Printing Data</h2>
-          <li>Bootstrap</li>
-          <li>Ruby on Rails</li>
-          <li>RSPec</li>
-          <li>CapyBara</li>
-          <li>Selenium</li>
+          <li class="light"><span class="Rectangle"></span> Frameworks</li>
+          <li class="experience">Bootstrap</li>
+          <li class="experience">Ruby on Rails</li>
+          <li class="experience">CapyBara</li>
+          <li class="experience">Selenium</li>
         </ul>
         
-        <ul class="dark"> 
-          <li class="light"><span class="Circle"></span> Skills</li> 
-        </ul>
+        
         <ul class="skill-list dark">
-          <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
+          <li class="light"><span class="Circle"></span> Skills</li> 
+            <li class="experience">Github</li>
+            <li class="experience">Codepen</li>
+            <li class="experience">Gitlab</li>
+            <li class="experience">Terminal</li>
         </ul>
       </div>
       </div>        
   </section> 
 
-  <section class="form-footer">
-    <div class="form-description">
-      <h2>I'm always interested in hearing about new projects, 
-      so if you'd like to chat please get in touch.</h2>
-    </div>
+<section class="form-footer">
+  <h2 class="form-description" >I'm always interested in hearing about new projects, 
+  so if you'd like to chat please get in touch.
+  </h2>
+    
 
-    <form action="https://formspree.io/f/mqkwbdbj" method="POST">
-      <ul class="form-section">
-        <li class="name">
-          <label for="name"></label>
-          <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
-        </li>
-
-        <li class="name">
-          <label for="mail"></label>
-          <input type="email" id="mail" name="user_email" placeholder="Email address" required>
-        </li>
-
-        <li class="text">
-          <label for="msg"></label>
-          <textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
+  <form action="https://formspree.io/f/mqkwbdbj" method="POST">
+    <ul class="form-section">
+      <li class="name">
+        <label for="name"></label>
+        <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
       </li>
 
-      <li class="button-submit">
-          <button type="submit">Get in touch</button>
+      <li class="name">
+        <label for="mail"></label>
+        <input type="email" id="mail" name="user_email" placeholder="Email address" required>
       </li>
 
-      </ul>
+      <li class="text">
+        <label for="msg"></label>
+        <textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
+      </li>
+
+      <li>
+          <button class="button-submit" type="submit">Get in touch</button>
+      </li>
+
+    </ul>
 
     </form>

```

Comment:
Reduce the width of the form as shown below 

#### Original design
![right4](https://user-images.githubusercontent.com/58376020/122623630-30acb580-d09d-11eb-8988-b58655719b60.png)

#### Your project
![wrong4](https://user-images.githubusercontent.com/58376020/122623654-3a361d80-d09d-11eb-928b-41c1affec4bd.png)